src/components/FlowRunTimeline.vue

```diff
@@ -309,11 +309,45 @@
     api.flowRuns.getFlowRuns,
     subFlowRunsSubscriptionFilter,
   )
+
+  const subFlowsSubscriptionFilter = computed<Parameters<typeof api.flows.getFlows> | null>(() => {
+    if (!subFlowRunsSubscription.response || subFlowRunsSubscription.response.length < 1) {
+      return null
+    }
+
+    const subFlowIds = subFlowRunsSubscription.response
+      .map((flowRun) => flowRun.flowId)
+
+    return [
+      {
+        flows: {
+          id: subFlowIds,
+        },
+      },
+    ]
+  })
+
+  const subFlowSubscription = useSubscriptionWithDependencies(
+    api.flows.getFlows,
+    subFlowsSubscriptionFilter,
+  )
```

Comment:
We should make a `useFlows` composition for this.

Reply:
You [resolved](https://github.com/PrefectHQ/prefect-ui-library/pull/1307) this one for me!